NitroxClient/Communication/Packets/Processors/DisconnectProcessor.cs

```diff
@@ -16,6 +17,7 @@ public DisconnectProcessor(PlayerGameObjectManager playerGameObjectManager)
         public override void Process(Disconnect disconnect)
         {
             playerGameObjectManager.RemovePlayerGameObject(disconnect.PlayerId);
+            ClientLogger.WriteLine(disconnect.PlayerId + " disconnected");
```

Comment:
I'm not sure we should mix messages that need to always go to the player.  If someone had logging turned off then the player would not see this message.  Would recommend having a new method for that.